static/sass/_sidebar-nav.scss

```diff
@@ -0,0 +1,277 @@
+          height: 1.875em;
+          position: absolute;
+          right: $gutter;
+          top: $gutter / 2;
+          width: $button-width;
+        }
+
+        &.collapsed {
+          cursor: pointer;
+          padding-right: $gutter + $button-width;
+
+          & > ul {
+            display: none;
+          }
+
+          &:after {
+            content: '+';
+          }
+        }
+
+        &.expanded {
+          cursor: pointer;
+          
+          & > ul {
+            display: block;
+          }
+          &:after {
+            content: '-';
+          }
+        }
+      }
+    }
+
+    & > ul {
+      // Hide the bottom-most border
+      // only if it's pushing right up against the footer
+      // so we don't get an ugly double-border effect
+      @media only screen and (min-width: $breakpoint-medium) {
+        margin-bottom: -1px;
+      }
+
+      & > li {
+        padding-bottom: $gutter / 2;
+        padding-top: $gutter / 2;
+        
+        @media only screen and (min-width: $breakpoint-medium) {
+          & > a {
+            font-size: 1rem;
+          }
+        }
+
+        & > ul {
+          & > li {
+            padding-left: $gutter;
+            padding-right: 0;
+          }
+        }
+
+        &:after {
+          background-position: center center;
+          background-repeat: no-repeat;
+          height: 1.5rem;
+        }
+        &.collapsed:after {
+          background-image: url('https://assets.ubuntu.com/v1/04d2075a-chevron_down.svg');
+          content: '';
+        }
+        &.expanded:after {
+          background-image: url('https://assets.ubuntu.com/v1/f3f43376-chevron_up.svg');
+          content: '';
+        }
+      }
+    }
+
+    // Top level sections get bottom padding.
+    .section {
+      padding: 0 0 20px;
+      // Subsections & collapsed sections get no padding.
+      &.collapsed, .section {
+        padding: 0;
+      }
+    }
+
+    .versions-nav {
+      padding: 1em 0;
+
+      .unavailable {
+        color: $color-dark;
+        text-decoration: line-through;
+      }
+
+      .label {
+        display: block;
+        margin-bottom: .5rem;
+      }
+
+      ul {
+        border-right: 1px solid $color-mid-light;
+        display: inline;
+        padding-bottom: .2rem;
+
+        li {
+          display: inline-block;
+          margin-right: .3rem;
+
+          a {
+            border-bottom: 1px solid $color-dark;
+            line-height: 1.1;
+            margin: 0;
+            padding: 0;
+            text-decoration: none;
+          }
+
+          strong {
+            border-bottom: 1px solid $color-brand;
+          }
+        }
+      }
+
+      .stable {
+        border-bottom: 1px solid $color-dark;
+        margin-left: .5rem;
+        text-decoration: none;
+      }
+    }
+  }
+
+  .note {
+    background-color: #fdf6e7;
+    border: 1px solid #eca918;
+    border-radius: 2px;
+    color: inherit;
+    padding: 20px;
+  }
```

Comment:
Let remove and use Vanilla notifications: https://github.com/ubuntudesign/vanilla-framework/blob/master/scss/_patterns_notifications.scss